index.Rmd

```diff
@@ -22,16 +22,25 @@ Leaflet is designed with simplicity, performance and usability in mind. It works
 Leaflet is one of the most popular open-source JavaScript libraries for interactive maps. It’s used by websites ranging from The New York Times and The Washington Post to GitHub and Flickr, as well as GIS specialists like OpenStreetMap, Mapbox, and CartoDB.
 
 This R package makes it easy to integrate and control Leaflet maps in R.
+
+A cheatsheet for this package is here: https://ugoproto.github.io/ugo_r_doc/leaflet-cheat-sheet.pdf
+
 ```{r echo=FALSE, warning=FALSE}
 #Example of a simple map
 
 #Check if leaflet is installed, if not install it
-list.of.packages <- c("leaflet", "DT")
```

Comment:
Clever solution to a problem that can be avoided with the 'require' function - checks if a package is installed and only installs it if it is not already.